docs/src/modules/components/LocalizationTable.js

```diff
@@ -3,35 +3,65 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { styled } from '@mui/material/styles';
+import { green } from '@mui/material/colors';
 import Link from 'docs/src/modules/components/Link';
 import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
 
 const Root = styled('div')(({ theme }) => ({
   position: 'relative',
   overflow: 'hidden',
-  width: 'fit-content',
+  minWidth: '100%',
+  display: 'flex',
+  justifyContent: 'center',
   paddingLeft: theme.spacing(1),
   paddingRight: theme.spacing(1),
   borderRadius: 5,
+  fontWeight: 600,
```

Comment:
```suggestion
```
Is there a reason to increase the font weight? 

<table>
<tr>
 <td>With
 <td>Without
<tr>
 <td><img src="https://github.com/mui/mui-x/assets/45398769/98c6f2bc-aee8-43af-842b-8c447682ad53"/>
 <td><img src="https://github.com/mui/mui-x/assets/45398769/39ff8c2d-773f-4aef-a2d3-fb4d773e9812" />
</table>

Reply:
It increases legibility, but the tradeoff is that it may make the progress bar more prominent. It depends on what is more important in this case and how much attention we want to bring to this section.

Reply:
I am leaning towards keeping a bolder `fontWeight`, but @alexfauquette what do you think about the importance of this section?

Reply:
> what do you think about the importance of this section?

I assume people are looking first for their locale so they will focus on the 3 left columns, and then check if something is missing. So the two `fontWeight` make sens